src/global/_web3modal.css

```diff
@@ -1,20 +1,17 @@
-div.web3modal-modal-lightbox,
-div.walletconnect-qrcode__base {
-  background: var(--background-body-transparent);
-  backdrop-filter: blur(3px);
+div.web3modal-modal-lightbox {
   overflow-x: hidden;
   overflow-y: auto;
   -webkit-overflow-scrolling: touch;
   animation: fadeIn 0.2s ease-out backwards;
 }
 
 div.web3modal-modal-card {
-  border-radius: var(--border-radius);
+  border-radius: 0;
   padding: var(--spacer);
   margin: var(--spacer) auto;
   max-width: 100%;
-  border: 1px solid var(--border-color);
-  box-shadow: 0 6px 15px 0 var(--box-shadow-color);
+  box-shadow: 0 20px 25px -5px rgb(0 0 0 / 10%),
+    0 10px 10px -5px rgb(0 0 0 / 4%);
```

Comment:
see the previous comments. As we are already using this 3 times I feel like it should be standardized.